src/main/webapp/jsp/instructorStudentRecords.jsp

```diff
@@ -205,6 +205,7 @@
                                 <div id="commentTextEdit<%=commentIdx%>" style="display:none;">
                                     <div class="form-group form-inline">
                                         <div class="form-group text-muted">
+                                            Comment about <%=InstructorStudentRecordsPageData.sanitizeForHtml(data.student.name)%>:<br>
```

Comment:
inconsistent?
1 br for instructorStudentRecords.jsp, but 2 for instructorFeedbackResults*.jsp